### PR TITLE
[hotfix]onboard_template

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Fix the problem that onboard create node with templates doesn't work. Closes https://github.com/CenterForOpenScience/osf.io/issues/3124

### CODE REVIEW

File: website/static/js/projectCreator.js [hunk 1]
```diff
@@ -68,11 +68,17 @@ function ProjectCreatorViewModel(params) {
 
     self.serialize = function() {
         var category = self.category();
+        var template;
+        if($osf.isIE()){
+            template = $('.createNodeTemplates')[3].value;
```
Comment (sloria) [inline website/static/js/projectCreator.js]: Can you add a comment explaining this?

**Comment by sloria** [comment]
I am still seeing the original issue when templating the project that Sarah mentioned.

File: website/static/js/projectCreator.js (hunk 1 above)
Comment (chennan47) [inline website/static/js/projectCreator.js]: addressed in  ccc9420

**Comment by chennan47** [comment]
@sloria rdy for another pass

**Comment by sloria** [comment]
@chennan47 This PR seems to contain a number of unrelated changes; can you look into this?

**Comment by chennan47** [comment]
@sloria Now it should be clear. rdy for review

File: website/static/js/osfHelpers.js [hunk 2]
```diff
@@ -658,6 +658,14 @@ var _confirmationString = function() {
 };
 
 /**
+*  Helper function to judge if the user browser is IE
+*/
+var isIE = function(userAgent) {
+  userAgent = userAgent || navigator.userAgent;
```
Comment (sloria) [inline website/static/js/osfHelpers.js]: > Use 4 spaces for indentation

https://cosdev.readthedocs.org/en/latest/style_guides/javascript.html#style

Will fix this when I merge.

**Pull request merged**